lib/rules/media-feature-range-notation/__tests__/index.mjs

```diff
@@ -110,6 +110,152 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['context'],
+	computeReplacementText: true,
+
+	accept: [
+		{
+			code: '@media {}',
+			description: 'empty media query',
+		},
+		{
+			code: '@media () {}',
+			description: 'empty media feature',
+		},
+		{
+			code: '@media screen {}',
+			description: 'keyword',
+		},
+		{
+			code: '@media (color) {}',
+			description: 'range type media feature in boolean context',
+		},
+		{
+			code: '@media (color > 0) {}',
+			description: 'range type media feature in non-boolean context',
+		},
+		{
+			code: '@media (pointer: fine) {}',
+			description: 'discrete type media feature',
+		},
+		{
+			code: '@media (width >= 1px) {}',
+			description: 'range type media feature in context notation',
+		},
+		{
+			code: '@media screen and (width >= 1px) {}',
+			description: 'range type media feature in context notation with keyword',
+		},
+		{
+			code: '@media not print, (width >= 1px) {}',
+			description: 'range type media feature in context notation in media query list',
+		},
+		{
+			code: '@media (1px <= width <= 2px) {}',
+			description: 'range type media feature in context notation with two values',
+		},
+	],
+
+	reject: [
+		{
+			code: '@media (min-width: 1px) {}',
+			description: 'range type media feature in prefix notation',
+			message: messages.expected('context'),
+			line: 1,
+			column: 8,
+			endLine: 1,
+			endColumn: 24,
+			fix: {
+				range: {
+					start: { line: 1, column: 9 },
+					end: { line: 1, column: 19 },
+				},
+				text: 'width >=',
+			},
```

Comment:
Really excited about this result.

I think it is really neat that the reported range and text is so narrow while requiring so little effort from plugin authors.